app/config/AppConfig.scala

```diff
@@ -103,10 +104,7 @@ class AppConfig @Inject()(
   lazy val defaultFeatureStatus: features.FeatureStatus.Value =
     FeatureStatus.withName(loadConfig(feature2Key(Feature.default)))
 
-  def wcoMetadataMapper(): WcoMetadataMapper with WcoMetadataMappingStrategy = {
-    logger.warn("Using WcoMetadataJavaMappingStrategy as the WCO-DEC mapper")
-    new WcoMetadataMapper with WcoMetadataJavaMappingStrategy
-  }
+  def wcoMetadataMapper(): WcoMetadataMapper = new WcoMetadataMapper
```

Comment:
Do you use this method somewhere? Is this method important? It just creates `WcoMetadataMapper`, so you can inline it.